.github/workflows/checkAndSubmitAddonMetadata.yml

```
@@ -219,44 +224,24 @@ jobs:
         # If no valid result is produced, i.e. no add-on id is found, jq exits with status 4.
         # If there is a different error, a different non-zero exit code is used.
         exit $? # Exit with the same exit code as jq
-
+        continue-on-error: true
     - name: Create discussion for add-on id
       id: createDiscussion
-      if: ${{ steps.checkDiscussion.outcome }} == 'failure'
+      if: (${{ success() }} || ${{ failure() }}) && (${{ steps.checkDiscussion.outcome }} == 'failure' || 
```

Shouldn't this just be? The syntax is wrong here regardless (should end in `)` not `||`.

```suggestion
      if: ${{ steps.checkDiscussion.outcome }} == 'failure'
```